### ISSUE TITLE
[Feature] Option to Move bars to bottom

### ISSUE THREAD

**Issue by mburucuyapy** [rafaelmardojai/firefox-gnome-theme#861]
Any chance to have the option to move the tabs and the address bar to the bottom of the display? This will be a bit more comfortable on tablets and other mobile devices.

**Comment by CleoMenezesJr** [comment]
I think we could consider doing something similar to what GNOME Web does.

**Comment by rafaelmardojai** [comment]
This is kind of in the same page of this old issue: #133.

I personally don't see the point of a preference like this in desktop mode, and the theme is kind of broken in small screens so moving everything to the bottom wont make everything magically fit.

**Comment by CleoMenezesJr** [comment]
Yes, from that point of view it really is something to consider.
